{
  "uid" : "8512c7b59f2aad31",
  "name" : "TC 01.03.20 correctness of the \"Hoodies & Sweatshirts\" link of the \"Men\" section",
  "fullName" : "tests.test_header.TestHeader#test_tc_01_03_20_correctness_of_hoodies_men_section_link",
  "historyId" : "f7ad61cb96dc4e8979ad11622bafed7f",
  "time" : {
    "start" : 1697334065968,
    "stop" : 1697334071321,
    "duration" : 5353
  },
  "description" : "Check 'Hoodies & Sweatshirts' subsection link click in 'Men' section redirects to the         Hoodies & Sweatshirts-men page and the 'Hoodies & Sweatshirts' header is displayed",
  "descriptionHtml" : "<p>Check 'Hoodies &amp; Sweatshirts' subsection link click in 'Men' section redirects to the         Hoodies &amp; Sweatshirts-men page and the 'Hoodies &amp; Sweatshirts' header is displayed</p>\n",
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5587a10cc6b3 <unknown>\n#1 0x5587a0da21e7 <unknown>\n#2 0x5587a0de9566 <unknown>\n#3 0x5587a0de9651 <unknown>\n#4 0x5587a0e26a74 <unknown>\n#5 0x5587a0e0b0ed <unknown>\n#6 0x5587a0e24322 <unknown>\n#7 0x5587a0e0ae93 <unknown>\n#8 0x5587a0ddd934 <unknown>\n#9 0x5587a0dde71e <unknown>\n#10 0x5587a1091cb8 <unknown>\n#11 0x5587a1095bf0 <unknown>\n#12 0x5587a10a019c <unknown>\n#13 0x5587a1096808 <unknown>\n#14 0x5587a106327f <unknown>\n#15 0x5587a10bae88 <unknown>\n#16 0x5587a10bb059 <unknown>\n#17 0x5587a10cb843 <unknown>\n#18 0x7f4c43e94b43 <unknown>",
  "statusTrace" : "self = <test_header.TestHeader object at 0x7fe0af35e710>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"43ce525aa56086ed16bfe772189745ea\")>\n\n    @allure.title('TC 01.03.20 correctness of the \"Hoodies & Sweatshirts\" link of the \"Men\" section')\n    def test_tc_01_03_20_correctness_of_hoodies_men_section_link(self, driver):\n        \"\"\"Check 'Hoodies & Sweatshirts' subsection link click in 'Men' section redirects to the \\\n        Hoodies & Sweatshirts-men page and the 'Hoodies & Sweatshirts' header is displayed\"\"\"\n        page = HeaderPage(driver, MAIN_PAGE_URL)\n        page.open()\n>       page.check_redirection_of_hoodies_subsection_of_men_section_link()\n\ntests/test_header.py:361: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/header_page.py:276: in check_redirection_of_hoodies_subsection_of_men_section_link\n    self.action_move_to_element(self.element_is_visible(self.header_locators.MEN_SECTION))\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"43ce525aa56086ed16bfe772189745ea\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fe0b0a03060>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5587a10cc6b3 <unknown>\nE       #1 0x5587a0da21e7 <unknown>\nE       #2 0x5587a0de9566 <unknown>\nE       #3 0x5587a0de9651 <unknown>\nE       #4 0x5587a0e26a74 <unknown>\nE       #5 0x5587a0e0b0ed <unknown>\nE       #6 0x5587a0e24322 <unknown>\nE       #7 0x5587a0e0ae93 <unknown>\nE       #8 0x5587a0ddd934 <unknown>\nE       #9 0x5587a0dde71e <unknown>\nE       #10 0x5587a1091cb8 <unknown>\nE       #11 0x5587a1095bf0 <unknown>\nE       #12 0x5587a10a019c <unknown>\nE       #13 0x5587a1096808 <unknown>\nE       #14 0x5587a106327f <unknown>\nE       #15 0x5587a10bae88 <unknown>\nE       #16 0x5587a10bb059 <unknown>\nE       #17 0x5587a10cb843 <unknown>\nE       #18 0x7f4c43e94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1697332188525,
      "stop" : 1697332188634,
      "duration" : 109
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder",
    "time" : {
      "start" : 1697332188634,
      "stop" : 1697332188634,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1697334064921,
      "stop" : 1697334065968,
      "duration" : 1047
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "Check 'Hoodies & Sweatshirts' subsection link click in 'Men' section redirects to the         Hoodies & Sweatshirts-men page and the 'Hoodies & Sweatshirts' header is displayed",
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5587a10cc6b3 <unknown>\n#1 0x5587a0da21e7 <unknown>\n#2 0x5587a0de9566 <unknown>\n#3 0x5587a0de9651 <unknown>\n#4 0x5587a0e26a74 <unknown>\n#5 0x5587a0e0b0ed <unknown>\n#6 0x5587a0e24322 <unknown>\n#7 0x5587a0e0ae93 <unknown>\n#8 0x5587a0ddd934 <unknown>\n#9 0x5587a0dde71e <unknown>\n#10 0x5587a1091cb8 <unknown>\n#11 0x5587a1095bf0 <unknown>\n#12 0x5587a10a019c <unknown>\n#13 0x5587a1096808 <unknown>\n#14 0x5587a106327f <unknown>\n#15 0x5587a10bae88 <unknown>\n#16 0x5587a10bb059 <unknown>\n#17 0x5587a10cb843 <unknown>\n#18 0x7f4c43e94b43 <unknown>",
    "statusTrace" : "self = <test_header.TestHeader object at 0x7fe0af35e710>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"43ce525aa56086ed16bfe772189745ea\")>\n\n    @allure.title('TC 01.03.20 correctness of the \"Hoodies & Sweatshirts\" link of the \"Men\" section')\n    def test_tc_01_03_20_correctness_of_hoodies_men_section_link(self, driver):\n        \"\"\"Check 'Hoodies & Sweatshirts' subsection link click in 'Men' section redirects to the \\\n        Hoodies & Sweatshirts-men page and the 'Hoodies & Sweatshirts' header is displayed\"\"\"\n        page = HeaderPage(driver, MAIN_PAGE_URL)\n        page.open()\n>       page.check_redirection_of_hoodies_subsection_of_men_section_link()\n\ntests/test_header.py:361: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/header_page.py:276: in check_redirection_of_hoodies_subsection_of_men_section_link\n    self.action_move_to_element(self.element_is_visible(self.header_locators.MEN_SECTION))\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"43ce525aa56086ed16bfe772189745ea\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fe0b0a03060>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5587a10cc6b3 <unknown>\nE       #1 0x5587a0da21e7 <unknown>\nE       #2 0x5587a0de9566 <unknown>\nE       #3 0x5587a0de9651 <unknown>\nE       #4 0x5587a0e26a74 <unknown>\nE       #5 0x5587a0e0b0ed <unknown>\nE       #6 0x5587a0e24322 <unknown>\nE       #7 0x5587a0e0ae93 <unknown>\nE       #8 0x5587a0ddd934 <unknown>\nE       #9 0x5587a0dde71e <unknown>\nE       #10 0x5587a1091cb8 <unknown>\nE       #11 0x5587a1095bf0 <unknown>\nE       #12 0x5587a10a019c <unknown>\nE       #13 0x5587a1096808 <unknown>\nE       #14 0x5587a106327f <unknown>\nE       #15 0x5587a10bae88 <unknown>\nE       #16 0x5587a10bb059 <unknown>\nE       #17 0x5587a10cb843 <unknown>\nE       #18 0x7f4c43e94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Open a browser",
      "time" : {
        "start" : 1697334065968,
        "stop" : 1697334066196,
        "duration" : 228
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Check redirection of the \"Hoodies & Sweatshirts\" subsection of \"Men\" section link",
      "time" : {
        "start" : 1697334066196,
        "stop" : 1697334071321,
        "duration" : 5125
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5587a10cc6b3 <unknown>\n#1 0x5587a0da21e7 <unknown>\n#2 0x5587a0de9566 <unknown>\n#3 0x5587a0de9651 <unknown>\n#4 0x5587a0e26a74 <unknown>\n#5 0x5587a0e0b0ed <unknown>\n#6 0x5587a0e24322 <unknown>\n#7 0x5587a0e0ae93 <unknown>\n#8 0x5587a0ddd934 <unknown>\n#9 0x5587a0dde71e <unknown>\n#10 0x5587a1091cb8 <unknown>\n#11 0x5587a1095bf0 <unknown>\n#12 0x5587a10a019c <unknown>\n#13 0x5587a1096808 <unknown>\n#14 0x5587a106327f <unknown>\n#15 0x5587a10bae88 <unknown>\n#16 0x5587a10bb059 <unknown>\n#17 0x5587a10cb843 <unknown>\n#18 0x7f4c43e94b43 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/header_page.py\", line 276, in check_redirection_of_hoodies_subsection_of_men_section_link\n    self.action_move_to_element(self.element_is_visible(self.header_locators.MEN_SECTION))\n                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Find a visible element",
        "time" : {
          "start" : 1697334066196,
          "stop" : 1697334071320,
          "duration" : 5124
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5587a10cc6b3 <unknown>\n#1 0x5587a0da21e7 <unknown>\n#2 0x5587a0de9566 <unknown>\n#3 0x5587a0de9651 <unknown>\n#4 0x5587a0e26a74 <unknown>\n#5 0x5587a0e0b0ed <unknown>\n#6 0x5587a0e24322 <unknown>\n#7 0x5587a0e0ae93 <unknown>\n#8 0x5587a0ddd934 <unknown>\n#9 0x5587a0dde71e <unknown>\n#10 0x5587a1091cb8 <unknown>\n#11 0x5587a1095bf0 <unknown>\n#12 0x5587a10a019c <unknown>\n#13 0x5587a1096808 <unknown>\n#14 0x5587a106327f <unknown>\n#15 0x5587a10bae88 <unknown>\n#16 0x5587a10bb059 <unknown>\n#17 0x5587a10cb843 <unknown>\n#18 0x7f4c43e94b43 <unknown>\n\n",
        "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ {
          "name" : "Find a present element",
          "time" : {
            "start" : 1697334066196,
            "stop" : 1697334071320,
            "duration" : 5124
          },
          "status" : "broken",
          "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5587a10cc6b3 <unknown>\n#1 0x5587a0da21e7 <unknown>\n#2 0x5587a0de9566 <unknown>\n#3 0x5587a0de9651 <unknown>\n#4 0x5587a0e26a74 <unknown>\n#5 0x5587a0e0b0ed <unknown>\n#6 0x5587a0e24322 <unknown>\n#7 0x5587a0e0ae93 <unknown>\n#8 0x5587a0ddd934 <unknown>\n#9 0x5587a0dde71e <unknown>\n#10 0x5587a1091cb8 <unknown>\n#11 0x5587a1095bf0 <unknown>\n#12 0x5587a10a019c <unknown>\n#13 0x5587a1096808 <unknown>\n#14 0x5587a106327f <unknown>\n#15 0x5587a10bae88 <unknown>\n#16 0x5587a10bb059 <unknown>\n#17 0x5587a10cb843 <unknown>\n#18 0x7f4c43e94b43 <unknown>\n\n",
          "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "locator",
            "value" : "('id', 'ui-id-5')"
          }, {
            "name" : "timeout",
            "value" : "5"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : true,
          "hasContent" : true
        } ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "locator",
          "value" : "('id', 'ui-id-5')"
        }, {
          "name" : "timeout",
          "value" : "10"
        } ],
        "stepsCount" : 1,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 2,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "44d04ed0b143bad4",
      "name" : "Screenshot",
      "source" : "44d04ed0b143bad4.png",
      "type" : "image/png",
      "size" : 14907
    }, {
      "uid" : "5eb2ba6582445fc4",
      "name" : "HTML source",
      "source" : "5eb2ba6582445fc4.html",
      "type" : "text/html",
      "size" : 571
    } ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "clear_allure_result_folder::0",
    "time" : {
      "start" : 1697335158583,
      "stop" : 1697335158584,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver::0",
    "time" : {
      "start" : 1697334071512,
      "stop" : 1697334071574,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Header"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_header"
  }, {
    "name" : "subSuite",
    "value" : "TestHeader"
  }, {
    "name" : "host",
    "value" : "fv-az163-423"
  }, {
    "name" : "thread",
    "value" : "3041-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_header"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "8512c7b59f2aad31.json",
  "parameterValues" : [ ]
}